packages/eas-cli/src/commands/build/index.ts

```diff
@@ -211,6 +207,7 @@ async function ensureProjectConfiguredAsync(projectDir: string): Promise<void> {
         'Build process requires clean git working tree, please commit all your changes and run `eas build` again'
       );
     }
+    return exp;
```

Comment:
I think it would be cleaner to just read config again here instead of modifying configureAsync, but I don't have a strong opinion on that